test/unit/test___main__.py

```diff
@@ -1,234 +0,0 @@
-import os
```

Comment:
I vote for leaving the tests in the repo.

Reply:
These tests call Runner directly as part of the unit test suite... that is antithetical to what unit tests are supposed to do.

Reply:
I prefer misclassified tests to no tests.